src/app/business/profile/profileCard/profile-card.component.ts

```diff
@@ -147,7 +151,7 @@ export class ProfileCardComponent implements OnInit {
             this.isAdministrator = false;
         }
         this.chartDatas = {
-            labels: ['Unused Capacity', 'Used Capacity'],
+            labels: ['Unused Capacity (GB)', 'Used Capacity (GB)'],
```

Comment:
Is GB the highest unit we convert and display?

Reply:
This is for profile @kumarashit . It shows the pool capacity that is assigned, which is fixed to 10 GB.

Reply:
Is it fixed at 10GB?

Reply:
Yes.

Reply:
@kumarashit After checking with @PravinRanjan10 I have concluded that the response is returned in GB for all drivers. 
This will work across the board for all drivers as of now. 
If there is a change we will update accordingly.
Changing the response of the pool capacity API is outside the scope of this PR.
Please review and approve.